docs/source/2019MemHeat.rst

```diff
@@ -8,7 +8,7 @@
 2019 Memorial Day Heat Wave
 =====================================
 
-A heat wave occurs on the weekend of 2019 Memorial Day across the Southeast U.S. 
+A heat wave occured on the weekend of 2019 Memorial Day across the Southeast U.S. 
```

Comment:
here it is ok, the heat wave really only occurred on that weekend. Note the spelling, though.